doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
-Using a standard interface for IO means that workload doesn't need to run any proprietary software drivers for specific hardware vendors and the implementation of that workload is completely agnostics of the hardware used.
+<!--Add diagram from Mark Contrell showing the Gap in the Open Source projects here.
+<p align="center"><img src="../figures/newfigure.jpg" alt="Gaps in SDN Open Source Projects" title="Gaps in SDN Open Source Projects"/></p>
+-->
 
-**Figure 4** below shows the typical components of a para-virtualized interface: 
-- frontEnd driver: The frontEnd driver is an off-the-shelf driver that runs on the workload.
-- backEnd driver: runs on the Hypervisor and is responsible of bridging standard communications coming from applications to a hardware specific ones. 
+<a name="1.3"></a>
+## 1.3 Project Goals and Purpose
+The goal of the task force is to develop a robust infrastructure model and a limited discrete set of architectures built on that model that can be tested and validated for use across the  entire member community. The community, which is made up of a cross section of global operators and supporting vendors alike, was created to support the development, deployment and management of NFV applications faster and more easily.  
 
-This nature of this disaggregation is what gives the para-virtualized interfaces the flexibility that makes them favourable in a virtualized environment.
+All of this had led to a growing awareness of the need to develop more open models and validation mechanisms to bring the most value to telco operators as well as vendors, by agreeing on a standard set of infrastructure profiles to use for the underlying infrastructure to support VNF applications across the industry and telecom community at large. To achieve this goal, the NFVI environment needs to be fully abstracted via APIs and other mechanisms to the VNFs so that both developers of the VNF applications and the operators managing the environments can benefit from the flexibility that the disaggregation of the underlying infrastructure offers.
 
-The downside of para-virtualization interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+The next step after the NFVI Reference Model has been identified and developed is to take the general model, which is purposely designed to be able to be applied to a number of technologies, and apply it to a discrete number of concrete and ultimately deployable Reference Architecture platforms. The intention is to chose the reference architectures carefully so that there will only be a small set of architectures that meets the specific requirements for supporting NFV and Telecom specific applications. Per the principles laid out in the Reference Model documentation, the Reference Architectures need to meet the following criteria as much as is practical:
```

Comment:
Given the discussion in #1168 can I suggest changing `the NFVI Reference Model` to `the Reference Model`?

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/04a20ae0d5b700b5d6f90e8b24035a71170efc9e